gptel.el

```diff
@@ -3,7 +3,7 @@
 ;; Copyright (C) 2023  Karthik Chikmagalur
 
 ;; Author: Karthik Chikmagalur <karthik.chikmagalur@gmail.com>
-;; Version: 0.8.6
+;; Version: 999.git
```

Comment:
Please revert this change, the version stays unchanged.

Reply:
@karthink Fixed with commit 3a748f5. Sorry, didn't mean to commit that originally. That's my hack to use `package-install-file` to ensure an installed dev branch supersedes the melpa branch.